### PR TITLE
Improved nodes +2 new nodes

### PR DESCRIPTION
**Pull request by knowledgenude**
* Improved Get Visible and Set Visible nodes. 

* Added Children Checkbox to Set Visible node as requested in: https://github.com/armory3d/armory/issues/1558 (thanks to @QuantumCoderQC [here](https://forums.armory3d.org/t/help-with-arrays/4207/5))

* Added Get Object Offscreen node.

* Added Get Rigid Body Data node with the possibility to get if an object has rigid body.

Updated test file: [visible_test.zip](https://github.com/armory3d/armory/files/5214725/visible_test.zip)

Preview: 
screenshot1

### CODE REVIEW

**Comment by knowledgenude** [comment]
I am not understanding these conflicts in LN_get_visible and LN_set_visible.

File: blender/arm/logicnode/object/LN_get_visible.py [hunk 1]
```diff
@@ -1,12 +1,15 @@
 from arm.logicnode.arm_nodes import *
 
 class GetVisibleNode(ArmLogicTreeNode):
-    """Get visible node"""
+    """Get Visible node"""
```
Comment (MoritzBrueckner) [inline blender/arm/logicnode/object/LN_get_visible.py:4]: To me it looks like this line is responsible for the conflicts, the GetVisible node docstring in current master doesn't end with "node" but here it is removed. You can try to locally merge the master branch into your branch to resolve the conflicts.

File: blender/arm/logicnode/object/LN_get_visible.py [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
     bl_idname = 'LNGetVisibleNode'
     bl_label = 'Get Visible'
+    bl_icon = 'NONE'
 
     def init(self, context):
-        self.add_input('ArmNodeSocketObject', 'Object')
```
Comment (MoritzBrueckner) [inline blender/arm/logicnode/object/LN_get_visible.py:9]: The same goes for this, https://github.com/armory3d/armory/pull/1860 introduced a `super()` call in `init()` that might be conflicting. Same solution as my other comment.

Comment (knowledgenude) [reply blender/arm/logicnode/object/LN_get_visible.py:9]: @MoritzBrueckner, okay. I didn't notice this new line there! Thank you :+1:

Comment (knowledgenude) [reply blender/arm/logicnode/object/LN_get_visible.py:9]: I really don't know what is wrong here. I've added `super(NodeNameNode, self).init(context)` line in all files  but nothing solves the problem. I will  give up until i start using git trough terminal. I will post the files in the forum, this way someone can send the PR. Thank you anyway!

Comment (MoritzBrueckner) [reply blender/arm/logicnode/object/LN_get_visible.py:9]: Do you have a GUI client for Git like Fork or Github Desktop? They make resolving conflicts a lot easier. But if you want I can create a PR for you, hopefully even with your commits so you get the credit :)

Comment (knowledgenude) [reply blender/arm/logicnode/object/LN_get_visible.py:9]: All this time i had used the Github site to do everything, but i always end with a lot of commits spammed. It is very limited. I will try to use terminal and test how it works in an own repository until i get familiar to it, but also there is some cool Guis to test, i just wanna one that i can do the things more "cleaner" :+1: 

Feel free to send a PR for me and thank you!

**Pull request closed without merging**